.github/dependabot.yml

```diff
@@ -27,3 +27,17 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+    # Ignore gcp dependencies
+    - dependency-name: "com.google.api:gax*"
+    - dependency-name: "com.google.cloud.bigdataoss:*"
+    - dependency-name: "com.google.cloud:google-cloud-spanner*"
+    - dependency-name: "com.google.code.gson:*"
+    - dependency-name: "com.google.oauth-client:*"
+    - dependency-name: "io.grpc:grpc-protobuf*"
+    - dependency-name: "com.google.guava:*"
+    - dependency-name: "com.google.protobuf:*"
+    - dependency-name: "com.google.cloud.datastore:datastore-v1-proto-client"
+    - dependency-name: "com.google.cloud:google-cloud-pubsub*"
+    - dependency-name: "com.google.cloud:libraries-bom"
+    - dependency-name: "com.google.api.grpc:proto-google-cloud-*"
```

Comment:
Can you also add `com.google.api.grpc:grpc-*` (Notice the difference "proto-" and "grpc-" prefix)

Can you also add `com.google.apis:google-api-services-*`?

Reply:
Done!